docs/source/scancode-toolkit/tutorials/how_to_add_a_post_scan_plugin.rst

```diff
@@ -0,0 +1,290 @@
+.. _how_to_add_post_scan_plugin:
+
+Add A Post-Scan Plugin
+======================
+
+Automatic vs. Manual Installation
+---------------------------------
+
+Automatic
+^^^^^^^^^
+
+Some post-scan plugins are installed when ScanCode itself is installed, e.g., the
+:ref:`license_policy_plugin`, whose code is located here::
+
+    https://github.com/nexB/scancode-toolkit/blob/develop/src/licensedcode/plugin_license_policy.py
+
+These plugins do not require any additional installation steps and can be used as soon as ScanCode
+is up and running.
+
+Manual
+^^^^^^
+
+ScanCode is also designed to use post-scan plugins that must be installed separately from the
+installation of ScanCode. The code for this sort of plugin is located here::
+
+    https://github.com/nexB/scancode-toolkit/tree/develop/plugins/
+
+This wiki page will focus on manually-installed post-scan plugins.
+
+Example Post-Scan Plugin: Hello ScanCode
+----------------------------------------
+
+To illustrate the creation of a simple post-scan plugin, we'll create a hypothetical plugin named
+``Hello ScanCode``, which will print ``Hello ScanCode!`` in your terminal after you've run a scan.
+Your command will look like something like this::
+
+    scancode -i -n 2 <path to target codebase> --hello --json <path to JSON output file>
+
+We'll start by creating three folders:
+
+1. Top-level folder -- ``/scancode-hello/``
+2. 2nd-level folder -- ``/src/``
+3. 3rd-level folder -- ``/hello_scancode/``
+
+1. Top-level folder -- ``/scancode-hello/``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- In the ``/scancode-toolkit/plugins/`` directory, add a folder with a relevant name, e.g.,
+  ``scancode-hello``. This folder will hold all of your plugin code.
+
+- Inside the ``/scancode-hello/`` folder you'll need to add a folder named ``src`` and 7 files.
+
+1. ``/src/`` -- This folder will contain your primary Python code and is discussed in more detail
+   in the following section.
+
+The 7 Files are:
+
+1. ``.gitignore`` -- See, e.g.,
+   `/plugins/scancode-ignore-binaries/.gitignore <https://github.com/nexB/scancode-toolkit/blob/develop/plugins/scancode-ignore-binaries/.gitignore>`_
+
+::
+
+    /build/
+    /dist/
+
+2. ``apache-2.0.LICENSE`` -- See, e.g.,
+   `/plugins/scancode-ignore-binaries/apache-2.0.LICENSE <https://github.com/nexB/scancode-toolkit/blob/develop/plugins/scancode-ignore-binaries/apache-2.0.LICENSE>`_
+
+3. ``MANIFEST.in``
+
+::
+
+    graft src
+
+    include setup.py
+    include setup.cfg
+    include .gitignore
+    include README.md
+    include MANIFEST.in
+    include NOTICE
+    include apache-2.0.LICENSE
+
+    global-exclude *.py[co] __pycache__ *.*~
+
+4. ``NOTICE`` -- See, e.g.,
+   `/plugins/scancode-ignore-binaries/NOTICE <https://github.com/nexB/scancode-toolkit/blob/develop/plugins/scancode-ignore-binaries/NOTICE>`__
```

Comment:
Let's be simple for contributed plugins and eschew a notice

Reply:
@pombredanne Could you please explain what you mean here?

Reply:
I mean we may not want a notice... but now that you said it was ported from the wiki, we can update that later.... so nothing to do for you for now.